src/uikit.cs

```diff
@@ -4267,8 +4255,8 @@ interface UICollectionViewDelegate : UIScrollViewDelegate {
 		UITargetedPreview GetPreviewForDismissingContextMenu (UICollectionView collectionView, UIContextMenuConfiguration configuration);
 
 		[NoWatch, NoTV, iOS (13,0)]
-		[Export ("collectionView:willCommitMenuWithAnimator:")]
-		void WillCommitMenu (UICollectionView collectionView, IUIContextMenuInteractionCommitAnimating animator);
+		[Export ("collectionView:willPerformPreviewActionForMenuWithConfiguration:animator:")]
+		void WillPerformPreviewActionForMenu (UICollectionView collectionView, UIContextMenuConfiguration configuration, IUIContextMenuInteractionCommitAnimating animator);
```

Comment:
Not sure the `ForMenu` adds much value here.

Reply:
agreed, it's not a direct `UIMenu` argument - but the signature makes it obvious

Reply:
Fixing!

```suggestion
		void WillPerformPreviewAction (UICollectionView collectionView, UIContextMenuConfiguration configuration, IUIContextMenuInteractionCommitAnimating animator);
```